MonoGame.Framework.Content.Pipeline/Processors/MaterialProcessor.cs

```diff
@@ -173,8 +173,10 @@ public override MaterialContent Process(MaterialContent input, ContentProcessorC
 
             // Build custom effects
             var effectMaterial = input as EffectMaterialContent;
-            if (effectMaterial != null)
+            if (effectMaterial != null && effectMaterial.CompiledEffect == null)
```

Comment:
This is not really necessary, but allows a custom processor to set a CompiledEffect directly.